src/MySQL/Driver.php

```diff
@@ -0,0 +1,14 @@
+<?php
+
+namespace atk4\dsql\MySQL;
+
+use atk4\dsql\Connection;
+
+class Driver extends Connection
+{
+    public $driverType = 'mysql';
```

Comment:
What is the actual purpose of driverType, when we can ceck if the class is instanceof? Should be probably removed.

Reply:
Refactored a bit more https://github.com/atk4/dsql/tree/refactor/organize-folder-structure-driver

WDYT?

Reply:
@georgehristov 

1. huge 👍 for splitting this PR. In the future, it is really better to isolate larger changes into more isolated oned, good job!
2. is dsql/218 the last part?
3. here are some comments to which seems to be not addressed

at least:
- about the `driverType` - is seems fully redundant for and I think we should test `instanceof Engine/Connection` to the right type (also much more refactorable/usable)
- in the new PR you use still `driver` word
- is it guaranteed that `create` does only create the underlaying driver and does not `connect`? (as with connect - I count with DB ping/issues, and with create - I expect no connection error/exception)